test/system/README.md

```diff
@@ -1,43 +1,24 @@
 # System tests
 
 These tests are built with BATS (Bash Automated Testing System). They are
-strongly influenced by libpod project.
+strongly influenced by the [libpod](https://github.com/containers/libpod) project.
```

Comment:
I took the liberty to split these cosmetics out into https://github.com/containers/toolbox/pull/377